src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -10,17 +32,32 @@
             <span>{{ createdOnTime }}</span>
         </div>
         <div class="c-ne__content">
-            <div :id="entry.id"
-                 class="c-ne__text"
-                 tabindex="0"
-                 :class="{ 'c-ne__input' : !readOnly }"
-                 :contenteditable="!readOnly"
-                 @blur="updateEntryValue($event)"
-                 @keydown.enter.exact.prevent
-                 @keyup.enter.exact.prevent="forceBlur($event)"
-                 v-text="entry.text"
-            >
-            </div>
+            <template v-if="readOnly && result">
+                <div
+                    :id="entry.id"
+                    class="c-ne__text highlight"
+                    tabindex="0"
+                >
+                    <TextHighlight
+                        :text="result.metadata.entryHit ? entry.text : `[ no result for '${result.metadata.originalSearchText}' in entry ]`"
```

Comment:
we can create new computed property or method.

Reply:
Ah, yes. Definitely the way to go.  Will do!